server/controllers/tickets.js

```diff
@@ -100,6 +127,28 @@ module.exports.getOneTicket = (req, res) => {
     });
 };
 
+/** WRITE TEST FOR THIS ROUTE **/
+module.exports.getUserTicketsByBoard = (req, res) => {
+  if (helper.checkUndefined(req.query.user_id, req.query.github_handle, req.query.board_id, req.query.api_key)) {
+    res.status(400).send('one of parameters from client is undefined');
+    return;
+  }
+  
+  var userHandle = req.query.github_handle;
+  var boardId = req.query.board_id;
+  dbhelper.getTicketsByUserHandleAndBoard(userHandle, boardId)
```

Comment:
Without converting to integer with parseInt(boardId), there's a chance that dbhelper will confuse the argument as a string and fail. I've had those failures before which is why I do parseInt on anything that is supposed to be an integer when passing into a dbhelper function.